client/src/components/app.jsx

```diff
@@ -3,6 +3,7 @@ import axios from 'axios';
 import ReviewList from './reviewList';
 import Ratings from './ratings';
 import Search from './search';
+import BackToAllReviews from './backToAllReviews';
 
 export default class App extends React.Component {
```

Comment:
Can you export default like this?  I'm not sure, but maybe put the export default at the bottom for consistency.

Reply:
Okay, cool.  Thanks for letting me know.